### ISSUE TITLE
Priority with python-scrapyd-api

### ISSUE THREAD

**Issue by vokhuongcse0604** [djm/python-scrapyd-api#9]
I want to set priority for my spider.
I done when set on scrapyd: 
curl http://localhost:6800/schedule.json -d project=myproject -d spider=somespider -d priority=1
How can i do like that on python-scrapyd-api?
Thanks.

**Comment by djm** [comment]
@vokhuongcse0604 

Looks like priority was a non-documented feature until recently.

The code in [scrapyd](https://github.com/scrapy/scrapyd/blob/master/scrapyd/spiderqueue.py#L16) just pops the priority kwarg off the arguments sent, so this can be done with our API with no code changes:

```
scrapyd.schedule('myproject', 'somespider', priority=1.1)
```

Priority should be an integer or a float.


---


Having said no code changes: I will update the docs to state this, so I'll leave this ticket open for now.

**Comment by Digenis** [comment]
I documented it in 1.1.1 after https://github.com/scrapy/scrapyd/issues/144#issuecomment-218963899
If you use priorities, make sure you checkout this pitfall https://github.com/scrapy/scrapyd/issues/187
Things should change in scrapyd-1.3

**Comment by djm** [comment]
@Digenis Yup, that's what I meant by poorly worded "until recently". Thank you very much for doing it and that helpful extra comment.
